doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+    "\n",
+    "* [1. Detecting drift](#drift_detection)\n",
+    "    * [Hypothesis testing](#hypothesis)\n",
+    "    * [Dimension reduction](#dimension_reduction)\n",
+    "    * [Simple example](#example)\n",
+    "  \n",
+    "* [2. Learned drift detection](#learned)\n",
+    "\n",
+    "* [3. Online detection](#online)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 0. What is drift? <a class=\"anchor\" id=\"what_is_drift\"></a>\n",
+    "\n",
+    "Although powerful, modern machine learning models can be sensitive. Seemingly subtle changes in a data distribution can destroy the performance of otherwise state-of-the art models, which can be especially problematic when ML models are deployed in production. Typically, ML models are tested on held out data in order to estimate their furture performance. Crucially, this assumes that the *process* underlying the input data $\\mathbf{X}$ and output data $\\mathbf{Y}$ remains constant. \n",
+    "\n",
+    "*Drift* is said to occur when the process underlying $\\mathbf{X}$ and $\\mathbf{Y}$ during deployment differs from the process that generated the training data. In this case, we can no longer expect the model's performance during deployment to match that observed on held out training data. If a *ground truth* is available during deployment, *supervised drift detection* can be performed, with the model's predictive performance monitored directly. However, in many deployment scenarios, such as the binary classification example below, a ground truth is not available and *unsupervised drift detection* methods are required.\n",
+    "\n",
+    "![drift in deployment](images/drift_deployment.png) \n",
+    "\n",
+    "To explore the different types of drift, consider the common scenario where we have input data $\\mathbf{X}$ and output data $\\mathbf{Y}$, jointly distributed according to $P(\\mathbf{X},\\mathbf{Y})$, and a model $f: x \\mapsto y$ trained on training/reference data with distribution $P_{ref}(\\mathbf{X},\\mathbf{Y})$. *Drift* is said to have occured when $P(\\mathbf{X},\\mathbf{Y}) \\ne P_{ref}(\\mathbf{X},\\mathbf{Y})$. Writing the joint distribution as\n",
+    "\n",
+    "$$\n",
+    "P(\\mathbf{X},\\mathbf{Y}) = P(\\mathbf{Y}|\\mathbf{X})P(\\mathbf{X}) = P(\\mathbf{X}|\\mathbf{Y})P(\\mathbf{Y}),\n",
+    "$$\n",
+    "\n",
+    "we can categorise drift into a number of types:\n",
+    "\n",
+    "- **Covariate drift**: Also referred to as input drift, this occurs when $P(\\mathbf{Y}|\\mathbf{X})$ = $P_{ref}(\\mathbf{Y}|\\mathbf{X})$, but the distribution of the input data has shifted $P(\\mathbf{X}) \\ne P_{ref}(\\mathbf{X})$. In a mortgage risk classification task, an example of this would be if at inference, applications for properties with more bedrooms than the properties present in the training data were considered. The model may now be extrapolating, potentially resulting in unreliable predictions. \n",
```

Comment:
Consider changing "at inference" with eg "at test time" given the ambiguous nature of the usage of inference.